src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -190,44 +161,35 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
           </a>
         )
       }
-      ContentIcon={
-        myProf && (
-          <FormControlLabel
-            control={
-              <Switch
-                onChange={handleChangeMobilePhonePrivacy}
-                color="secondary"
-                checked={!isMobilePhonePrivate}
-              />
-            }
-            label={isMobilePhonePrivate ? 'Private' : 'Public'}
-            labelPlacement="bottom"
-            disabled={!isOnline}
-          />
-        )
-      }
+      ContentIcon={myProf && UpdateUserPrivacy(profile.AD_Username, ['MobilePhone'])}
       privateInfo={isMobilePhonePrivate}
       myProf={myProf}
     />
   ) : null;
 
   let streetAddr = profile.HomeStreet2 ? <span>{profile.HomeStreet2},&nbsp;</span> : null;
 
+  let combineHomeLocation =
+    profile.Country === 'United States of America' || !profile.Country
```

Comment:
```suggestion
    profile.Country === 'United States of America' || !profile.Country
```
Second time this line comes up. Potentially better to extract it into its own variable that is used both here and in the `home` variable below